src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -524,7 +526,8 @@ def reset_ocsp_response_cache_uri(ocsp_response_cache_uri) -> None:
         # len(OCSP_RESPONSE_VALIDATION_CACHE) is thread-safe, however, we do not want to
         # block for logging purpose, thus using len(OCSP_RESPONSE_VALIDATION_CACHE._cache) here.
         logger.debug(
-            "OCSP_VALIDATION_CACHE size: %s", len(OCSP_RESPONSE_VALIDATION_CACHE._cache)
+            "OCSP_VALIDATION_CACHE size: %s",
```

Comment:
`%s` is for strings, `%i`, or `%u` might be more appropiate
```suggestion
            "OCSP_VALIDATION_CACHE size: %u",
```